client/components/forms/docs/example.jsx

```diff
@@ -188,7 +189,7 @@ var FormFields = React.createClass( {
 					</FormButtonsBar>
 				</Card>
 
-				<Card>
+				<Card id="form-section">
```

Comment:
Any particular reason why this `id` was added? I don't see it reference anywhere in our code/assets. Maybe it's a leftover from a merge?